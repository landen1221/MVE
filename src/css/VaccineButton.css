.MuiButton-contained {
  color: green;
  background-color: royalblue;
  margin-left: 10px;
  margin-right: 20px;
  font-size: 500px;
}
