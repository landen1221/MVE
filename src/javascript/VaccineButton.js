import Button from "@material-ui/core/Button";
import "../css/VaccineButton.css";

const VaccineButton = ({ vaccine }) => {
  return (
    <>
      <Button variant="contained" color="primary" id="VaccineButton">
        {vaccine}
      </Button>
    </>
  );
};

export default VaccineButton;
