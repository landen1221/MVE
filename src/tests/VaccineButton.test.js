import ReactDOM from "react-dom";
import VaccineButton from "../javascript/VaccineButton";
import { render } from "@testing-library/react";

it("renders without crashing", () => {
  const div = document.createElement("div");
  ReactDOM.render(<VaccineButton></VaccineButton>, div);
});

it("matches snapshot", function () {
  const { asFragment } = render(<VaccineButton></VaccineButton>);
  expect(asFragment()).toMatchSnapshot();
});
