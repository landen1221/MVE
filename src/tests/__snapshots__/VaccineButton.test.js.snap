// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <button
    class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary"
    id="VaccineButton"
    tabindex="0"
    type="button"
  >
    <span
      class="MuiButton-label"
    />
    <span
      class="MuiTouchRipple-root"
    />
  </button>
</DocumentFragment>
`;
